.github/workflows/android.yml

```diff
@@ -46,8 +46,7 @@ jobs:
       run: printenv | sort
 
     - name: Install dependencies
-      working-directory: ${{runner.workspace}}/TeamTalk5/Build
-      run: sudo make depend-ubuntu24-android
+      run: sudo apt-get install cmake doxygen junit4 ninja-build
```

Comment:
We don't know if the dependency list in Build/Makefile is correct if we don't test it.